helper_posix_test.go

```diff
@@ -149,6 +144,22 @@ func copyTestdataFileTo(src, dest string) [][]string {
 	}
 }
 
+func rawCopyTestdataFileTo(src, dest string) string {
+	sourcePath := filepath.Join(testdataDir, src)
+	return `"command": [
+    [
+      "mkdir",
+      "-p",
+      "`+ filepath.Dir(dest) +`"
+    ],
+    [
+      "cp",
+      "`+ sourcePath +`",
+      "`+ dest +`"
+    ]
+  ]`
```

Comment:
You updated `rawHelloGoodbye` to longer include `"command": ` but in `rawCopyTestdataFileTo` you are including `"command": `. Is that intentional? We should probably be consistent in whether it is included or not.

Reply:
Definitely not intentional...